plugins/catalog-backend-module-incremental-ingestion/src/router/routes.ts

```diff
@@ -253,12 +248,6 @@ export class IncrementalProviderRouter {
       },
     );
 
-    const middleware = MiddlewareFactory.create({
-      logger: this.logger,
-      config: this.config,
-    });
-    router.use(middleware.error());
```

Comment:
This is already redundant and we can get rid of it either way right?

Reply:
Except that when you do, you get less informative error output.